Formula/a/acervus.rb

```diff
@@ -0,0 +1,17 @@
+class Acervus < Formula
+  desc "CLI tool for interacting with Acervus Cloud"
+  homepage "https://github.com/paribu/acervus-cli"
+  url "https://github.com/paribu/acervus-cli/archive/refs/tags/v0.0.1.tar.gz"
+  sha256 "d02a6465aed995f4a18b289d7e16f203dd875cc2bd9db5ad2572bfd68754d175"
+  head "https://github.com/paribu/acervus-cli.git", branch: "main"
+
+  depends_on "go@1.20" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_match "acervus version 0.0.1", shell_output("#{bin}/acervus --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.